src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
The return statement here is returning a fixed cost for any number of days that are not 3, 6, or 7. Instead, the function should calculate the cost based on the number of days and apply discounts accordingly. The cost should be calculated by multiplying the number of days by the daily cost and then subtracting any applicable discounts.